tests/instrumentation/python_memcached_tests.py

```diff
@@ -42,7 +42,7 @@
 
 
 if "MEMCACHED_HOST" not in os.environ:
-    pytestmark.append(pytest.mark.skip("Skipping memchached tests, no MEMCACHED_HOST environment variable set"))
+    pytestmark.append(pytest.mark.skip("Skipping memcached tests, no MEMCACHED_HOST environment variable set"))
```

Comment:
:bow:

Reply:
I only noticed because I started from a copy-paste of those tests in the first place xD